src/objects/domain_obj.f90

```diff
@@ -1124,14 +1130,15 @@ subroutine setup_sleve(this, options)
 
                     endif
 
+                    warn_threshold_dz=0.75
```

Comment:
this is fine for now, ultimately someone might want to run a simulation with a surface layer 0.75m thick, though it should probably have a small dx for that case too.  Maybe this should search for `dz/dx < threshold` in the future?

Reply:
agreed